### DISCUSSION TITLE
Metaprogramming, iyakah? wah ngerinya. - Part 1

### DISCUSSION THREAD

**Discussion by giscus** [r17x/rin.rocks#38] [Q&A]
# Metaprogramming, iyakah? wah ngerinya. - Part 1

https://www.rin.rocks/posts/metaprogramming-1

**Comment by munirapp** [comment]
sekarang jadi 3 orang

![comments](https://img001.prntscr.com/file/img001/unC1I6L6RdaWCYQXNvPN8Q.png)
